### PR TITLE
Corrected error calculations for x1dsums

### PR DESCRIPTION
**Pull request by stscirij**
By correctly converting the error in counts to error in flux units.  Also removed pqfactor terms

### CODE REVIEW

**Review by chjohnson1 — APPROVED** [review]
For the normalizeSums change I verified that the units are correct for the new error values.  The units of the "conversion" variable should be erg A^-1 cm^-2 counts^-1 and the units for the error values should be counts s^-1.  Multiplying the error by the conversion should give a unit of flux as erg A^-1 cm^-2 s^-1.

It looks like the accumulateSums change is the removal of the "pqfactor", which was kind of a fudge factor anyway.

**Comment by Rplesha** [comment]
@chjohnson1 for the future, we shouldn't approve this until all of our tests have been run

File: calcos/fpavg.py [hunk 1]
```diff
@@ -1181,30 +1181,47 @@ def normalizeSums(self, data, sumweight):
...
+        nonzeroweight = np.where(data.field("dq_wgt") != 0.)
+        zeroweight = np.where(data.field("dq_wgt") == 0.)
         data.field("flux")[:] /= sumweight
-        data.field("error")[:] = np.sqrt(data.field("error")) / sumweight
         data.field("gross")[:] /= sumweight
-        data.field("net")[:] /= sumweight
         data.field("background")[:] /= sumweight
-        gcounts = data.field("gcounts")[:]
         variance = data.field("variance_flat") + data.field("variance_counts") + data.field("variance_bkg")
+#        variance[nonzeroweight] = variance[nonzeroweight] / data.field("dq_wgt")[nonzeroweight]
```
Comment (Rplesha) [inline calcos/fpavg.py:1199]: Do you want to delete this commented out line?

**Review by Rplesha — APPROVED** [review]
Approving this. There's one leftover commented line that you might want to delete, but it isn't hurting anything being left in

**Review by Rplesha — APPROVED** [review]
Looks good!

**Pull request merged**